pandas/src/period.pyx

```diff
@@ -1203,6 +1208,7 @@ class Period(_Period):
 
 def _ordinal_from_fields(year, month, quarter, day,
                          hour, minute, second, freq):
+    freq = Period._maybe_convert_freq(freq)
```

Comment:
this doesn't needed, as ``_ordinal_from_field`` is only called from here (after 1130th line)?

- https://github.com/pydata/pandas/blob/5f524d61fd336b850d34f13d5ffb2b6136073f21/pandas/src/period.pyx#L1152